<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSubject</class>
 <widget class="QWidget" name="EditSubject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QListView" name="classList"/>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="subjectName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Количество часов</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="hourCount">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Первое занятие</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="firstClassDate"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Даты занятий:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Добавление предмета:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="backwardButton">
     <property name="text">
      <string>Готово</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="audoDates">
     <property name="text">
      <string>Проставить автоматически</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backwardButton</sender>
   <signal>clicked()</signal>
   <receiver>EditSubject</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audoDates</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditSubject</receiver>
   <slot>setDatesState(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hourCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EditSubject</receiver>
   <slot>setHourCount(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goBack()</slot>
  <slot>setDatesState(int)</slot>
  <slot>setHourCount(int)</slot>
 </slots>
</ui>
